tidy3d/plugins/design/result.py

```diff
@@ -186,8 +188,27 @@ def to_dataframe(self) -> pandas.DataFrame:
             data.append(data_i)
 
         val_keys = list(self.value_as_dict(self.values[0])) if self.values else [""]
+
         columns = list(self.dims) + val_keys
 
+        if include_aux:
+            if self.aux_values is not None:
```

Comment:
```py
if include_aux and self.aux_values:
```
or consider just warning if `include_aux` but not `self.aux_values`? not sure the intended API.